#goal
sensor_msgs/PointCloud2 cloud_input
geometry_msgs/Point ignore_origin
std_msgs/Float32 ignore_start_angle
std_msgs/Float32 ignore_end_angle
---
#result
sensor_msgs/PointCloud2 table_points
geometry_msgs/Quaternion table_coefs
visualization_msgs/Marker table_bounding_box
sensor_msgs/PointCloud2[] objects
visualization_msgs/Marker[] object_bounding_boxes
---
#feedback
